ntoskrnl/mm/ARM3/section.c

```diff
@@ -2911,8 +2914,12 @@ MmMapViewOfArm3Section(IN PVOID SectionObject,
     ULONG ProtectionMask;
     NTSTATUS Status;
     ULONG64 CalculatedViewSize;
+
     PAGED_CODE();
 
+    /* Check for invalid inherit disposition */
+    // Let MiMapViewOfDataSection() check InheritDisposition value.
```

Comment:
Is this a TODO? What does that even mean?

Reply:
No. It means there "should" be the same `ASSERT` as new line 1291, but that would be redundant.